item.py

```diff
@@ -29,7 +30,7 @@ class BaseItem:
     """
```

Comment:
Shouldn't it be class BaseItem(AbstractItem) with no apply_passive function in the class for it to be treated as an abstract subclass automatically ? Or am I getting it wrong ?

Reply:
Abstractmethod makes it so that all subclasses have to have the apply passive method
We initialize it with a pass ( do nothing) so that we can always apply passive and only the items with a real one defind in the specific item class like serrated will have an effect